contracts/seed/Seed.sol

```diff
@@ -64,6 +64,7 @@ contract Seed {
     uint256 public feeRemainder; // Amount of seed tokens remaining for the fee
     uint256 public fundingCollected; // Amount of funding tokens collected by the seed contract.
     uint256 public fundingWithdrawn; // Amount of funding token withdrawn from the seed contract.
+    uint256 public totalFee; //Amount of all fee claimed when the seed was claimed.
```

Comment:
Can you rename it to feeClaimed pls, so it would be the same with seedClaimed?

Reply:
Done